tensai-fabric/src/main/java/dev/vmsa/tensai/fabric/mixins/MinecraftServerMixin.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of tensai, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) 2022 VMSA
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package dev.vmsa.tensai.fabric.mixins;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.server.MinecraftServer;
+
+import dev.vmsa.tensai.fabric.TensaiFabricEvents;
+
+@Mixin(MinecraftServer.class)
```

Comment:
Better use FabricAPI

Reply:
I didn't see Fabric API in ``build.gradle``. Do we want them to install Fabric API along with Tensai Fabric?

Reply:
For compatibility, yes. Fabric API is much more stable than Minecraft's

Reply:
Please note that Fabric API is heavily version-dependent.

https://github.com/FabricMC/fabric/blob/58bef3f2cf4beaa9ecbc5a96e568de8fa90e2485/src/main/resources/fabric.mod.json#L21-L25